go/chat/boxer.go

```diff
@@ -87,6 +87,15 @@ func (b *Boxer) makeErrorMessage(msg chat1.MessageBoxed, err UnboxingError) chat
 	})
 }
 
+func (b *Boxer) detectKBFSPermanentServerError(err error) bool {
+	// Banned folders are only detectable by the error string currently, hopefully
+	// we can do something better in the future.
+	if err.Error() == "Operations for this folder are temporarily throttled (error 2800)" {
```

Comment:
Can we get a real error type instead of checking a string?  Where does 2800 come from?  It's not in keybase/lib/status_codes.iced.